run-upgrade-tests.sh

```diff
@@ -85,7 +96,13 @@ else
     STORAGE_URL=file://$GNOCCHI_DATA
 fi
 
+# This downgrade of `numpy` is needed to enable the merge of PR
+# https://github.com/gnocchixyz/gnocchi/pull/1279, which is the PR that
+# introduces the support of numpy >= 1.24. After we merge it, we can remove
+# this downgrade here.
+pip install "numpy<1.24"
```

Comment:
Pin this separately in setup.cfg and then propose a commit following the changes required that unpins it.

Reply:
I am not sure I follow your idea of creating a new commit. It will not work if I do that in this PR.

Reply:
You can remove this entirely as it's not needed right?